packages/xod-client/src/index.js

```diff
@@ -75,6 +76,7 @@ export * from './popups/constants';
 export { lowercaseKebabMask } from './utils/inputFormatting';
 export { default as sanctuaryPropType } from './utils/sanctuaryPropType';
 export * from './core/urlActions';
+export * from './core/messages';
```

Comment:
I think in that particular case we loose context when consume API

```js
client.invalidUrlActionName(...) // <- I can’t guess what this thing does
```

Isn’t it a better idea to:

```js
// index.js
export * as Messages from './core/messages'

// App.jsx
client.Messages.invalidUrlActionName(actionName)
```

Reply:
Done!